resources/definitions/flyingbear_ghost_4s.def.json

```diff
@@ -17,7 +17,7 @@
 		"machine_name": 		{ "default_value": "Flying Bear Ghost 4S" },
 		"machine_width": 		{ "default_value": 255 },
 		"machine_depth": 		{ "default_value": 210 },
-		"machine_height": 		{ "default_value": 210 },
+		"machine_height": 		{ "default_value": 200 },
```

Comment:
Why are you reducing the height? The official build volume mentions the height as 210mm.

Reply:
Hi! I admire your attention to detail! 😊 

The default Z-coupler on "4S" is terrible. So, one of the most common upgrades for "4S" is a coupler like shown in the picture  ![image](https://user-images.githubusercontent.com/17224129/123157103-cc11a200-d472-11eb-974a-32ac1f52b4a6.png)

This coupler reduces the maximum Z-axis travel to a value between 200 and 205.

When the "5" came out, the manufacturer began to supply printers with a new coupler, identical to the one shown in the picture, with the same safe Z max. Despite the official information on the build volume, even if you max tighten bed springs, 210mm on Z could never be achieved. 

If you think the decision to reduce of height should be made and applied by the user, I will remove that change.

Reply:
That's perfectly reasonable then.